dom0/sd-whonix-template-files.sls

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
Since this is VMs that are not deleted in place, we should remove this configuration as part of `make clean`. I suggest we implement this logic in sd-clean-all https://github.com/freedomofpress/securedrop-workstation/blob/configure_logging_world/dom0/sd-clean-all.sls